docarray/array/storage/annlite/backend.py

```diff
@@ -25,6 +27,7 @@ class AnnliteConfig:
     ef_construction: Optional[int] = None
     ef_search: Optional[int] = None
     max_connection: Optional[int] = None
+    columns: Optional[List[Tuple[str, str]]] = None
```

Comment:
this snippet needs to be added also (in the constructor, I believe right before `self._config = config`):
```python
        if self._config.columns is None:
            self._config.columns = []
```